# Matrix of type [Reward, X, Y] with dimensions [nx3]
uint32 rows
float32[] data